cache/cache.go

```diff
@@ -115,6 +126,22 @@ func isMovedError(err error) bool {
 	return false
 }
 
+// return a expiration duration with a jitter added
+// The result is between expiration-jitter and expiration+jitter.
+// Negative value are never returned, 0 is used instead
```

Comment:
```suggestion
// Return an expiration duration with a jitter added, i.e the actual expiration is in the range [expiration - jitter, expiration + jitter].
// If no expiration was set or jitter > expiration, then any negative value will yield 0 instead.
```
Curious if `jitter` > `expiration` should even be allowed, seems to me it should be strictly lower or equal. Thoughts?

Reply:
Yes, it's likely not wanted and we should probably reject/warn for such "bad" configuration.

Because this function avoid returning negative number, it's not "broken" but it will result in odd behavior. At least the gain of a jitter add/subtract a random value it that the average expiration is `expiration`. This become wrong if `jitter` > `expiration`.